_scripts/instruction-widget/install.js

```diff
@@ -116,6 +120,8 @@ module.exports = function(context) {
     context.package = "certbot"
     if (context.webserver == "apache") {
       context.package = "python-certbot-apache";
+    } else if (context.webserver == "nginx") {
+      context.certonly = true;
```

Comment:
Yes, but it has an earlier version of Certbot which isn't ready to be exposed yet.